flink-clients/src/test/java/org/apache/flink/client/program/ClientTest.java

```diff
@@ -139,6 +140,69 @@ public void shouldSubmitToJobClient() {
 	}
 
 	/**
+	 * Tests that invalid detached mode programs fail.
+	 */
+	@Test
+	public void testDetachedMode() throws Exception{
+		jobManagerSystem.actorOf(Props.create(SuccessReturningActor.class), JobManager.JOB_MANAGER_NAME());
+		Client out = new Client(config);
+
+		try {
+			PackagedProgram prg = new PackagedProgram(TestExecuteTwice.class, "/dev/random", "/tmp");
```

Comment:
Actually, these arguments are not needed. I will simply remove them.
Also, in `textGetExecutionPlan` where these values are used, the locations are not actually accessed. Only plan is tested. So shouldn't be an issue.